### ISSUE TITLE
mouse pointer has wrong size after switch between hidpi and non-hidpi monitor

### ISSUE THREAD

**Issue by jmarino** [linuxmint/cinnamon#5958]
I have a hidpi 4k laptop (XPS 15) that I connect to an external monitor at work. The external monitor is an old 90ish dpi display. I don't do two screens at the same time, just one at a time. External monitor at work (lodpi) and laptop display anywhere else (hidpi).

Switching from hidpi to lodpi always leaves the mouse pointer too large.
Switching from lodpi to hipdi always leaves the mouse pointer too small.

The problem is fixed, in both cases, by running: cinnamon --replace &

**Comment by icarter09** [comment]
@jmarino are you still experiencing this issue?

**Comment by jmarino** [comment]
I haven't used that monitor setup in a while. I still have the same 4k laptop but now I also have a 4k monitor with the same resolution.

I do have an older low res monitor at home and I tested things last night. I confirm that the issue is not present anymore. However, when I switched to the low res monitor (with xrand) cinnamon did not adjust the interface scaling-factor, I seem to remember that cinnamon would do that automatically before. In any case, that is not related to this issue.
